pytest.ini

```diff
@@ -2,3 +2,4 @@
 testpaths = scico/test docs
 addopts = --doctest-glob="*rst"
 doctest_optionflags = NORMALIZE_WHITESPACE
+filterwarnings = ignore::DeprecationWarning:.*.compat
```

Comment:
This should ignore all DeprecationWarnings originating in the compat module. I guess we would try to further restrict it to a specific line in compat, but this should already be specific enough to avoid problems, don't you think?

Reply:
Ah, I wasn't aware that it would filter it encased in compat. I don't mind it this way in that case! Just was unsure if it went out of scope for a second there. Thanks for the knowledge bit